package.json

```diff
@@ -25,15 +25,19 @@
     "prettier": "2.5.1"
   },
   "resolutions": {
-    "ethers": "5.5.2",
+    "ethers": "5.6.2",
     "@ethersproject/bignumber": "5.5.0",
     "@ethersproject/providers": "5.5.1",
-    "@urql/core": "2.3.6",
-    "@urql/exchange-execute": "1.2.2",
-    "scrypt": "https://registry.yarnpkg.com/@favware/skip-dependency/-/skip-dependency-1.0.2.tgz"
+    "@urql/core": "2.4.4",
+    "@urql/exchange-execute": "1.2.2"
   },
-  "dependencies": {
-    "scrypt": "https://registry.yarnpkg.com/@favware/skip-dependency/-/skip-dependency-1.0.2.tgz"
```

Comment:
Did we ever determine the reason for this dependency skip?

Reply:
No, I couldn't find it being used anywhere so I simply removed.  I'm inclined to think that it was used for vector/connext payment channels before they were removed.